### PR TITLE
Add captcha instructions

### PR DESCRIPTION
**Pull request by robdekort**
Can be merged when both the Captcha and Tools addons have been updated.

### CODE REVIEW

File: docs/features/forms.md [hunk 1]
```diff
@@ -60,4 +60,8 @@ When you want to populate a hidden field with custom data, you can add the follo
  <div x-init="form.hidden_field_handle = 'value', form.validate('hidden_field_handle')"></div>
 ```
 
-This will make sure that your hidden fields gets populated with the data you want and added to the precognition request. When the value comes from Alpine and needs to be dynamic you can use `x-effect` instead. 
+This will make sure that your hidden fields gets populated with the data you want and added to the precognition request. When the value comes from Alpine and needs to be dynamic you can use `x-effect` instead.
+
+## Captcha
+
+You can use Aryey Raber's wonderful [Captcha addon](https://statamic.com/addons/aryeh-raber/captcha) together with your Peak form. In order to make this work, follow the addons instructions and add `x-data='@{"captcha-response": "" }'` to your Peak `form:create` tag in `resources/views/page_builder/_form.antlers.html`. Lastly, make sure you run the latest versions of both the Peak Tools and the Captcha addons.
```
Comment (aryehraber) [inline docs/features/forms.md]: @robdekort Nice! I like how simple this solution is for users 👌 My only request, there's a typo in my name, it's "Aryeh" 😄

Comment (robdekort) [reply docs/features/forms.md]: Darn! I even made sure to check this. Obviously I failed :-)

**Comment by aryehraber** [comment]
PRs submitted ✅ 
- https://github.com/aryehraber/statamic-captcha/pull/60
- https://github.com/studio1902/statamic-peak-tools/pull/23

I've merged the PR on Captcha itself, adding the new `{{ captcha:selector }}` tag, and I've created a PR for Peak Tools which adds the extra lines to automatically handle setting the captcha response token if it is set on the Statamic form (as per your doc update).

Let me know if anything else is needed from my end! Was great working together on this to make integration simpler for Peak users 🤝

**Comment by robdekort** [comment]
Woohooo. This is awesome. Was fun indeed. Thanks Aryeh!

**Pull request merged**